add_book.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
```

Comment:
Observations: The code does not handle the output format or response structure.

Recommendations: Consider returning responses in a structured format like JSON, especially if this is part of an API.

Example:
```php
header('Content-Type: application/json');
$response = ['status' => 'success', 'message' => 'Book added successfully!'];
echo json_encode($response);
```